jetty-deploy/src/test/resources/jetty-logging.properties

```diff
@@ -1,3 +1,4 @@
 org.eclipse.jetty.util.log.class=org.eclipse.jetty.util.log.StdErrLog
-org.eclipse.jetty.deploy.LEVEL=WARN
-org.eclipse.jetty.util.Scanner=WARN
+#org.eclipse.jetty.deploy.DeploymentTempDirTest.LEVEL=DEBUG
+#org.eclipse.jetty.deploy.LEVEL=DEBUG
+#org.eclipse.jetty.util.Scanner=DEBUG
```

Comment:
Does default INFO level output too much logging?

Reply:
No there is not that much logging on INFO level.